src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/ServiceBusTriggerBinding.cs

```diff
@@ -94,7 +95,8 @@ public string EntityPath
         public async Task<ITriggerData> BindAsync(object value, ValueBindingContext context)
         {
             BrokeredMessage message = value as BrokeredMessage;
-            if (message == null && !_converter.TryConvert(value, out message))
+            if (message == null && _converter != null && 
+                !_converter.TryConvert(value, out message))
```

Comment:
I know `_converter` shouldn't be null anymore -- but if it is, won't this just skip over the throw here now? I may be reading it wrong, too :-)